bokeh/charts/__init__.py

```diff
@@ -1,40 +1,4 @@
 from __future__ import absolute_import
 
-from ..util.dependencies import import_required
-import_required(
-    'pandas',
-    'The bokeh.charts interface requires Pandas (http://pandas.pydata.org) to be installed.'
-)
-
-# defaults and constants
-from ..plotting.helpers import DEFAULT_PALETTE; DEFAULT_PALETTE
-
-# main components
-from .chart import Chart, defaults
-
-# operations and attributes for users to input into Charts
-from .attributes import color, marker, cat
-from .operations import stack, blend
-from .stats import bins
-
-# builders
-from .builders.line_builder import Line
-from .builders.histogram_builder import Histogram
-from .builders.bar_builder import Bar
-from .builders.scatter_builder import Scatter
-from .builders.boxplot_builder import BoxPlot
-from .builders.step_builder import Step
-from .builders.timeseries_builder import TimeSeries
-from .builders.dot_builder import Dot
-from .builders.area_builder import Area
-from .builders.horizon_builder import Horizon
-from .builders.heatmap_builder import HeatMap
-from .builders.donut_builder import Donut
-from .builders.chord_builder import Chord
-
-# easy access to required bokeh components
-from ..models import ColumnDataSource; ColumnDataSource
-from ..io import curdoc, output_file, output_notebook, reset_output, save, show, gridplot
-
-# Silence pyflakes
-(curdoc, output_file, output_notebook, reset_output, save, show, gridplot)
+# TODO (bev) restore shim when bkcharts package is ready
+# from bkcharts import * # NOQA
```

Comment:
Should we add deprecation message at this point, pointing people to preferably use `bkcharts`?